client/src/QnA/AnswerListEntry.jsx

```diff
@@ -34,49 +34,43 @@ function AnswerListEntry({ answer, grabAnswers }) {
 
   const reportListener = (e) => {
     e.preventDefault();
-    axios.put('/qa/answers/report', { answer_id: answer.answer_id })
+    axios.put('/qa/answers/report', { answer_id: answer.id })
       .then((result) => {
         setReport(!report);
       })
       .then(() => {
-        grabAnswers();
+        grabQuestions();
       })
       .catch((err) => {
         console.log('Error reporting answer', err);
       });
   };
 
-  // Invoking grabAnswers in the promises seems to make it render faster? Also less errors.
-  // useEffect(() => {
-  //   grabAnswers();
-  // }, [ansHelpful]);
-
   return (
-    <div>
-      <div>{answer.body}</div>
-      <span>
-        by
-        {' '}
-        {answer.answerer_name}
-        {' '}
-        {formattedDate}
-        {' '}
-        Helpful?
-        {' '}
-        <button onClick={helpfulListener} type="button" disabled={ansHelpful}>
-          Yes
-          {' '}
-          (
-          {answer.helpfulness}
-          )
-        </button>
-        <button onClick={reportListener} type="button" disabled={report}>
-          {' '}
-          Report
-        </button>
-      </span>
+    <div className="answer-individual">
+      <div className="answer-body">{answer.body}</div>
       <div className="answer-photo-container">
-        {answer.photos.map((photo) => <Photos photo={photo} key={photo.id} />)}
+        {answer.photos.map((photo) => (
+          <Photos photo={photo} key={photo} />
+        ))}
+      </div>
+      <div className="answerer-info">
+        <div className="answer-username">
+          by
+          {' '}
+          {answer.answerer_name}
+        </div>
+        <div className="answer-date">{formattedDate}</div>
+        <div className="answer-buttons">
+          <button onClick={helpfulListener} type="button" disabled={ansHelpful}>
+            Yes (
+            {answer.helpfulness}
+            )
+          </button>
+          <button onClick={reportListener} type="button" disabled={report}>
+            Report
+          </button>
+        </div>
```

Comment:
I can see you restructured your tree for how questions and answers will be affected by CSS and it looks good! Much easier to follow.
Also nice error catching messages.